IEXSharp/Helper/ExecutorREST.cs

```diff
@@ -70,15 +71,25 @@ public ExecutorREST(HttpClient client, string publishableToken, string secretTok
 						return new IEXResponse<ReturnType>() { ErrorMessage = token["error"].ToString() };
 					}
 					else if (content.Equals("forbidden", StringComparison.InvariantCultureIgnoreCase)
```

Comment:
I think here rather than handle each error status code individually (there's quite a lot and codes can share different reasons - https://iexcloud.io/docs/api/#error-codes) we should be doing something like:

```csharp
if (responseContent.StatusCode == HttpStatusCode.OK)
{
	// Successful Request
}
else
{
	// Failed Request
	return new IEXResponse<ReturnType> { ErrorMessage = $"{responseContent.StatusCode} - {content}" };
}
```

Reply:
Thanks for posting that link - I had no idea that there were so many more possible error codes